packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -283,7 +283,7 @@ const validateZip: OnValidateFunction = (value, focused) => {
   } else if (value.length !== 5 && !focused) {
     valid = false;
     error = 'Zip code is too short';
-  } else if (value.length == 5) {
+  } else if (value.length === 5) {
```

Comment:
Huh, I thought we'd fixed that in another PR already. That lint warning's been around too long.  😞